libs/assigner_runner/CMakeLists.txt

```diff
@@ -22,7 +22,6 @@ target_link_libraries(zkEVMAssignerRunner
                         zkEVMJsonHelpers
                         zkEVMOutputArtifacts
                         Boost::log
-                        Boost::random
```

Comment:
Do we need random in line 16?

Reply:
Nope, we don't, my bad
Removed it in fixup commit